
export const Blogs = async (req, res, next) => {
    try {
        
    } catch (error) {
        res.status(500).json({
            success: false,
            message: ["Internal Server Error". error.message]
        })
    }
}

export const BlogCreate = async (req, res, next) => {
    try {
        
    } catch (error) {
        res.status(500).json({
            success: false,
            message: ["Internal Server Error". error.message]
        })
    }
}

export const BlogDetails = async (req, res, next) => {
    try {
        
    } catch (error) {
        res.status(500).json({
            success: false,
            message: ["Internal Server Error". error.message]
        })
    }
}

export const BlogEdit = async (req, res, next) => {
    try {
        
    } catch (error) {
        res.status(500).json({
            success: false,
            message: ["Internal Server Error". error.message]
        })
    }
}

export const BlogDelete = async (req, res, next) => {
    try {
        
    } catch (error) {
        res.status(500).json({
            success: false,
            message: ["Internal Server Error". error.message]
        })
    }
}

export const BlogLatest = async (req, res, next) => {
    try {
        
    } catch (error) {
        res.status(500).json({
            success: false,
            message: ["Internal Server Error". error.message]
        })
    }
}